Client/Frontend/Settings/SettingsNavigationController.swift

```diff
@@ -16,6 +16,13 @@ class SettingsNavigationController: UINavigationController {
     }
 
     override var preferredStatusBarStyle: UIStatusBarStyle {
+        if #available(iOS 13, *) {
+            if self.view.appearanceOverrideUserInterfaceStyle == .light || self.overrideUserInterfaceStyle == .light {
+                return .darkContent
+            } else {
```

Comment:
This is so minor imo, but it's worth mentioning just for learning purposes I guess.

```
if condition {
    return a
} else {
    return b
}

//can also be written as (without the else):

if condition {
    return a
}
return b
```